lib/demos/cupertino/cupertino_picker_demo.dart

```diff
@@ -27,24 +27,36 @@ class _CupertinoPickerDemoState extends State<CupertinoPickerDemo> {
   // Value that is shown in the date picker in dateAndTime mode.
```

Comment:
Since you changed the code in a demo file, please run:

```
flutter pub run grinder verify-code-segments
```

to update the code segments, as described here: https://github.com/flutter/gallery/#generating-localized-strings-and-highlighted-code-segments